src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java

```diff
@@ -1618,8 +1503,7 @@ public boolean verifyElasticsearchConnection() {
         return connected;
     }
 
-    @Scheduled(
-            fixedRate = 1800000 )
+    @Scheduled( fixedRate = 1800000 )
     public void verifyRunner() throws UnknownHostException {
         verifyElasticsearchConnection();
```

Comment:
I've always wondered if this is the recommended way of doing this.

Reply:
same.... I *think* this originally came from one of elasticsearch's getting started guides, but that was a very long time ago